addon/components/truncate-multiline.js

```diff
@@ -193,6 +194,38 @@ export default Ember.Component.extend(ResizeHandlerMixin, {
    */
   resizeOnInsert: false,
 
+  /**
+   * Helper function to determine if an attribute has changed.
+   * @private
+   * @param attrs {Object}
+   * @param name {String}
+   * @return {Boolean}
+   */
+  _didAttrChange(attrs, name) {
```

Comment:
Let's move this off of the component definition. It can simply be a helper function within scope.